### DISCUSSION TITLE
plotting problem for k35

### DISCUSSION THREAD

**Discussion by magicptop** [Chia-Network/chia-blockchain#7143] [Support]
Hello everyone... I am new user and I just meet the some problem! 
System plotting  stuck.....
anyone can help me ?
thank you!
![S__125632515](https://user-images.githubusercontent.com/87039867/124659586-b6bb5f80-ded7-11eb-9930-2bfa3edb5e95.jpg)

**Comment by loppefaaret** [comment]
it could be that you ran out of room on the drive you are using for plotting!

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
